README.md

```diff
@@ -1,66 +1,67 @@
 # The OpenAPI Initiative Style Guide
 
-How to (and how not to) refer to the OAI in meetups, interviews, casual conversations, the settling of bar bets, and for conference presentations. 
+How to (and how not to) refer to the **OpenAPI Initiative (OAI)** and the **OpenAPI Specification (OAS)** in meetups, interviews, casual conversations, the settling of bar bets, and for conference presentations. 
 
 ## The Specification Name and Usage
 
-**OpenAPI Specification** refers the name of the popular API description format. It contains ONE space and FIVE capital letters. Following the first instance, it may be referred to as:
+The OAS refers to the name of the popular API description format. It contains ONE space and FIVE capital letters. Following the first instance, it may be referred to as:
```

Comment:
Adding the omitted word "to" is helpful. Thanks.

I find it confusing to use "OAS" instead of the full form, "OpenAPI Specification." The full form _does_ contain one space and five capital letters; and it _does_ make sense to say that it may be abbreviated "following the first instance."

But the short form "OAS" _does not_ contain one space and five capital letters, and it _does not_ make sense to say that it can be referred to as "OAS" following the first instance.

Reply:
Regarding OAS, I did not make up that acronym or style.  It was already there.  When editing, I assumed the previous material was already reviewed and approved.  I was just trying to make the document less wordy and more readable.  By the way, most searching the internet for "OAS" and "Swagger" returns 89,000 results whereas "OpenAPI Specification" and "Swagger" only produce 67,500 results.  So, OAS is more popular, because people like acronyms rather than long titles.  This is especially true when reading a document.  However, I am not arguing, just stating the facts and also basing it on my 40 years of technical writing experience.  Ted, do you have the final say, or should we bring this up to the group to decide to change the naming?

Reply:
@clairestreb, I don't have the final say about anything. My company, RepreZen, is an OAI member, so I have a "vote" in these matters, but I think these style guidelines are under the purvue of Marketing working group, in which I am (usually) not very active. Maybe Neil can advise on the process. 

To clarify, I have no objection to using "OAS" instead of "OpenAPI Specification" in other contexts, only in this specific context.  If you read the sentence, it is making specific reference to the name "OpenAPI Specification," describing the precise type treatment of that name.

>  **OpenAPI Specification** refers the name of the popular API description format. It contains ONE space and FIVE capital letters.

I'm certainly not trying to make the argument that the long form "OpenAPI Specification" is somehow better than "OAS" and I never imagined that you "made up that acronym." :-)

I appreciate your efforts to improve on this content.

Reply:
Oh, I understand what you are saying now!  Sorry about that.  I just realized that ONE space and FIVE capital letters is mentioned in two places, so I will fix it and update my pull request.